substrate/frame/examples/src/lib.rs

```diff
@@ -17,24 +17,27 @@
 
 //! # FRAME Pallet Examples
 //!
-//! This crate contains examples of FRAME pallets. It is not intended to be used in production.
+//! This crate contains a collection of simple examples of FRAME pallets, demonstrating useful
+//! features in action. It is not intended to be used in production.
 //!
 //! ## Pallets
 //!
-//! - [**`pallet_example_basic`**](./basic): A simple example of a FRAME pallet demonstrating
-//! concepts, APIs and structures common to most FRAME runtimes.
+//! - [`pallet_example_basic`]: This pallet demonstrates concepts, APIs and structures common to
```

Comment:
Yes, these links seem to have been broken for a while https://github.com/paritytech/polkadot-sdk/pull/1554#discussion_r1327925822